PHPCompatibility/Tests/Syntax/RemovedCurlyBraceArrayAccessUnitTest.php

```diff
@@ -132,4 +132,14 @@ public function testNoViolationsInFileOnValidVersion()
         $file = $this->sniffFile(__FILE__, '7.3');
         $this->assertNoViolation($file);
     }
+
+    /**
+     * Ensure that the fixer produces expected output
+     *
+     * @return void
+     */
+    public function testFixer()
+    {
+        $this->assertFixerOutputMatches(str_replace('.php', '.inc', __FILE__));
```

Comment:
This works more by accident than by design without passing the `testVersion`... Also see #142.